helpers/lib_tasks_lint.py

```diff
@@ -229,13 +234,26 @@ def lint(  # type: ignore
     hlitauti.report_task()
     # Prepare the command line.
     lint_cmd_opts = []
-    # Add the file selection argument.
-    hdbg.dassert_eq(
+    # Enumerate file flags for assertion
```

Comment:
-> Count the command line options used to select files.

Also need a `.` at the end